src/include/common/types/types.h

```diff
@@ -54,6 +54,7 @@ KUZU_API enum DataTypeID : uint8_t {
     DATE = 25,
     TIMESTAMP = 26,
     INTERVAL = 27,
+    TENSOR = 28,
```

Comment:
Let's not add TENSOR, instead, I think the correct way is to replace LIST with two new data types: FIXED_LIST, VAR_LIST.
FIXED_LIST covers what TENSOR needs here.

Reply:
Changed to FIXED_LIST, VAR_LIST